language-server/modules/langserver-commons/src/main/java/org/ballerinalang/langserver/commons/service/spi/ExtendedLanguageServerService.java

```diff
@@ -35,10 +34,9 @@
     /**
      * Initialize callback for the service.
      *
-     * @param documentManager document manager
-     * @param langServer      language server
+     * @param langServer language server
      */
-    default void init(WorkspaceDocumentManager documentManager, LanguageServer langServer) {
+    default void init(LanguageServer langServer) {
```

Comment:
Earlier `WorkspaceDocumentManager` was the access point for the document. Thus, we had this parameter. Since now `WorkspaceManager` is the access point; shall we pass `WorkspaceManager` instead.

Reply:
Addressed with https://github.com/ballerina-platform/ballerina-lang/pull/27600